imap/jmap_blob.c

```diff
@@ -285,28 +283,6 @@ static int jmap_blob_copy(jmap_req_t *req)
         goto cleanup;
     }
 
-    char *srcinbox = mboxname_user_mbox(copy.from_account_id, NULL);
-    char *dstinbox = mboxname_user_mbox(req->accountid, NULL);
-    if (strcmp(srcinbox, dstinbox) < 0) {
-        srcnamespacelock = mboxname_usernamespacelock(srcinbox);
-        dstnamespacelock = mboxname_usernamespacelock(dstinbox);
-    }
-    else {
-        dstnamespacelock = mboxname_usernamespacelock(dstinbox);
-        srcnamespacelock = mboxname_usernamespacelock(srcinbox);
-    }
-    free(srcinbox);
-    free(dstinbox);
-
-    // now we can open the cstate
-    r = conversations_open_user(req->accountid, 0, &req->cstate);
-    if (r) {
-        syslog(LOG_ERR, "jmap_email_copy: can't open converstaions: %s",
-                        error_message(r));
-        jmap_error(req, jmap_server_error(r));
-        goto cleanup;
-    }
-
     /* Check if we can upload to toAccountId */
```

Comment:
While we are at it, we could get rid of this stale comment. There was an `toAccountId` argument in early drafts of  Blob/copy, but the final API uses `accountId` and `fromAccountId`